src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java

```diff
@@ -143,16 +143,22 @@ public List<InmateBasicDetails> getBasicInmateDetailsForOffenders(final Set<Stri
         log.info("getBasicInmateDetailsForOffenders, {} offenders, {} caseloads, canViewAllOffenders {}", offenders.size(), caseloads.size(), canViewAllOffenders);
 
         final var results = new ArrayList<InmateBasicDetails>();
+
         Lists.partition(Lists.newArrayList(offenders), maxBatchSize).forEach(offenderList ->
             results.addAll(
                 repository.getBasicInmateDetailsForOffenders(new HashSet<>(offenderList), canViewAllOffenders, caseloads, active)
                     .stream()
-                    .map(offender ->
-                        offender.toBuilder()
-                            .firstName(WordUtils.capitalizeFully(offender.getFirstName()))
-                            .middleName(WordUtils.capitalizeFully(offender.getMiddleName()))
-                            .lastName(WordUtils.capitalizeFully(offender.getLastName()))
-                            .build()
+                    .map(offender -> {
+                            var middleName = "";
+                            if (offender.getMiddleName() != null) {
+                                middleName = WordUtils.capitalizeFully(offender.getMiddleName()).trim();
+                            }
```

Comment:
Trim middle names as we'll be doing that with the new endpoint.

Reply:
bit naff having a mutable variable, might be better to just do
```
                                middleName = StringUtils.trim(WordUtils.capitalizeFully(offender.getMiddleName()));
```